### PR TITLE
Define and activate AWS 9.0.9 platform release

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/12440

### CODE REVIEW

**Comment by sslavic** [comment]
Tested 9.0.9 release definition with app upgrades only on giraffe:
- 9.0.9 cluster creation
- upgrading from 9.0.8

Like for aws 9.3.6 @calvix feel free to add and test aws-operator, calico, flatcar upgrade.

**Comment by calvix** [comment]
AWS operator 5.5.4 with the latest flatcar  is done,  changes pushed to release file as well

**Comment by calvix** [comment]
I will now work on the calico and than update release notes

**Comment by calvix** [comment]
found in when testing the release in the IC controller
```
E0731 08:19:07.554856       8 leaderelection.go:356] Failed to update lock: configmaps "nginx-ingress-controller-leader-nginx" is forbidden: User "system:serviceaccount:kube-system:nginx-ingress-controller" cannot update resource "configmaps" in API group "" in the namespace "kube-system"
```

**Comment by calvix** [comment]
my changes are done, release notes updated, you can go ahead @sslavic 

I tested cluster creation 9.0.9 and cluster update from 9.0.8

**Comment by sslavic** [comment]
perfect, thanks @calvix!

@cokiengchiara this one is ready to ship when you're OK with it.

**Comment by sslavic** [comment]
Working on the fix for https://github.com/giantswarm/releases/pull/381#issuecomment-666999180 in https://github.com/giantswarm/nginx-ingress-controller-app/pull/105

Great @calvix that you noticed it, thanks for reporting it!

**Comment by sslavic** [comment]
Working on the fix for https://github.com/giantswarm/releases/pull/381#issuecomment-666999180 in giantswarm/nginx-ingress-controller-app#105

File: aws/v9.0.9/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# :zap: Giant Swarm Release v9.0.9 for AWS :zap:
+
+This release updates managed apps to the latest releases, updates Calico to version `3.13.4` and Container Linux to `2512.2.1`.
```
Comment (cokiengchiara) [inline aws/v9.0.9/README.md:3]:
```suggestion
This release updates managed apps to the latest releases, Calico to version `3.13.4`, and Flatcar Container Linux to `2512.2.1`.
```

File: aws/v9.0.9/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### kube-state-metrics v1.9.7 ([Giant Swarm app v1.1.1](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#111---2020-07-22))
+
+- Updated kube-state-metrics version from 1.9.5 to 1.9.7. Check the [upstream changelog](https://github.com/kubernetes/kube-state-metrics/blob/master/CHANGELOG.md#v197--2020-05-24) for details on all changes.
+
+### metrics-server v0.3.6 ([Giant Swarm app v1.1.1](https://github.com/giantswarm/metrics-server-app/blob/master/CHANGELOG.md#111---2020-07-23))
+
+- Updated metrics-server version from 0.3.3 to 0.3.6. Check the [upstream changelog](https://github.com/kubernetes-sigs/metrics-server/releases) for details on all changes.
+
+### nginx-ingress-controller v0.34.1 ([Giant Swarm app v1.8.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#181---2020-07-28))
```
Comment (sslavic) [inline aws/v9.0.9/README.md:33]:
```suggestion
### nginx-ingress-controller v0.34.1 ([Giant Swarm app v1.8.2](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#181---2020-07-31))
```

File: aws/v9.0.9/release.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    name: kube-state-metrics
+    version: 1.1.1
+  - componentVersion: 0.3.6
+    name: metrics-server
+    version: 1.1.1
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 0.34.1
+    name: nginx-ingress-controller
+    version: 1.8.1
```
Comment (sslavic) [inline aws/v9.0.9/release.yaml:28]:
```suggestion
    version: 1.8.2
```

**Comment by sslavic** [comment]
Included nginx fix for leader election and retested on gauss:
- 9.0.9 cluster creation
- upgrade from 9.0.8

**Pull request merged**